### PR TITLE
fix: fixed errors due to arm instances

### PR DESCRIPTION
**Pull request by eggfoobar**
removed availability zone decleration to allow AWS to choose one which has our desired resource

added shell logic during instance creation to correctly create pv based off of device name and size

### CODE REVIEW

File: ci-operator/step-registry/openshift/microshift/infra/aws/ec2/openshift-microshift-infra-aws-ec2-commands.sh [hunk 1]
```diff
@@ -240,7 +239,16 @@ Resources:
...
+          sudo dnf install -y lvm2
+
+          # Print out the device name who's size is around 120GB, since that's the size we create.
+          # This is done due to x86 and arm being different in how device names are given
+          device_name=\$(lsblk -dnb | awk '{if (\$4>=128800000000 && \$4<=138800000000){printf "/dev/%s\n",\$1}}')
+
+          # NOTE: do not wrap script vars with {} since the cloudformation will see
+          # them as cloudformation vars instead.
+          sudo pvcreate "\$device_name"
+          sudo vgcreate rhel "\$device_name"
```
[the remark this answers is not in the document]
Comment (copejon) [reply ci-operator/step-registry/openshift/microshift/infra/aws/ec2/openshift-microshift-infra-aws-ec2-commands.sh:251]: True that the VG name can be arbitrary however the instances doesn't provision with LVM volumes by default.  So we still need the additional block device and still need to create a VG.

[the remark this answers is not in the document]
Comment (dhellmann) [reply ci-operator/step-registry/openshift/microshift/infra/aws/ec2/openshift-microshift-infra-aws-ec2-commands.sh:251]: OK, that's good to understand, thanks.

**Comment by eggfoobar** [comment]
/pj-rehearse

Seems like we are hitting the capacity problem since arm is failing at the provision step in that AZ. Trying another rehearsal to make sure x86 works, will try arm to see if it fails again in this AZ. I'll update the PR to include more subnets in different AZ's and hopefully that will allow us more options for selections.

**Comment by copejon** [comment]
The arm tests are not marked optional so we don't block up CI while we're debugging the issue.

**Comment by eggfoobar** [comment]
/pj-rehearse pull-ci-openshift-microshift-main-microshift-e2e-arm

**Comment by eggfoobar** [comment]
/pj-rehearse pull-ci-openshift-microshift-main-e2e-openshift-conformance-reduced-arm

**Pull request merged**